docs/hub/open_clip.md

```diff
@@ -0,0 +1,75 @@
+# Using OpenCLIP at Hugging Face
+
+[OpenCLIP](https://github.com/mlfoundations/open_clip) is the open-source library built to reproduce OpenAI's CLIP.
+
+## Exploring OpenCLIP on the Hub
+
+You can find OpenCLIP models by filtering at the left of the [models page](https://huggingface.co/models?library=open_clip&sort=trending).
+
+OpenCLIP models hosted on the Hub have a model card with useful information about the models. Thanks to OpenCLIP Hugging Face Hub integration, you can load OpenCLIP models with few lines of code from the Hub. You can also launch these models using [Inference Endpoints](https://huggingface.co/inference-endpoints).
+
+
+## Installation
+
+To get started, you can follow the [OpenCLIP installation guide](https://github.com/mlfoundations/open_clip#usage).
+You can also use the following one-line install through pip:
+
+```
+$ pip install open_clip_torch
+```
+
+## Using existing models
+
+All OpenCLIP models can easily be loaded from the Hub:
+
+```py
+import open_clip
+
+model, preprocess_train, preprocess_val = open_clip.create_model_and_transforms('hf-hub:laion/CLIP-ViT-g-14-laion2B-s12B-b42K')
+tokenizer = open_clip.get_tokenizer('hf-hub:laion/CLIP-ViT-g-14-laion2B-s12B-b42K')
+```
+
+Once loaded, you can encode the image and text to do zero-shot classification:
```

Comment:
Maybe link to the zero-shot image classification task page here

Reply:
Addressed this.